src/dev/impl/DevToys/ViewModels/Tools/Generators/HashGenerator/HashGeneratorToolViewModel.cs

```diff
@@ -161,7 +185,15 @@ private async Task<string> CalculateHashAsync(string alrogithmName, string text)
                 IBuffer buffer = CryptographicBuffer.ConvertStringToBinary(text, BinaryStringEncoding.Utf8);
                 buffer = algorithmProvider.HashData(buffer);
 
-                string? hash = CryptographicBuffer.EncodeToHexString(buffer);
```

Comment:
Hello, 
I see that you compare two different values, can you use a String.Equals to do that?
And also please avoid using string directly here and define an internal for example, you can see and example in the "JsonYamlToolViewModel"
Thank you

Reply:
I commited the changes, please check if now it's better

Reply:
It's seems good to me @veler do you want to check too?